src/recipes/editor-plugin/inspector-plugins.md

```diff
@@ -0,0 +1,342 @@
+<!--
+  ~ Copyright (c) godot-rust; Bromeon and contributors.
+  ~ This Source Code Form is subject to the terms of the Mozilla Public
+  ~ License, v. 2.0. If a copy of the MPL was not distributed with this
+  ~ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+-->
+
+# Inspector plugins
+
+The inspector dock allows you to create custom widgets to edit properties through plugins.
+This can be beneficial when working with custom datatypes and resources, although you can
+use the feature to change the inspector widgets for built-in types. You can design custom
+controls for specific properties, entire objects, and even separate controls associated
+with particular datatypes.
```

Comment:
The allowed line width is much more (around 140 chars). You don't have to fully use it, but you can use a bit longer lines than here :slightly_smiling_face:

Reply:
150 actually, but I don't really understand why it's so important

so should I change it?

Reply:
It's not too important, just helps consistency if it doesn't vary wildly between authors. So would be nice if it's not much work, but otherwise feel free to leave it.